### DISCUSSION TITLE
Error Call to a member function lang() on null

### DISCUSSION THREAD

**Discussion by uoe91** [statamic/cms#7381] [Troubleshooting]
Hello!
I launched my multisite online. But I am seeing an error
`Error
Call to a member function lang() on null`.

![image](https://user-images.githubusercontent.com/123072089/213417346-1d2d9770-0258-4293-af56-4b064d60837e.png)

Where can I look for a solution? I get the same error when I try to check the content of the page.

Thanks in advance!

**Comment by duncanmcclean** [comment]
What does your `config/statamic/sites.php` file look like?

**Reply by uoe91** [reply]
`<?php

return [

    /*
    |--------------------------------------------------------------------------
    | Sites
    |--------------------------------------------------------------------------
    |
    | Each site should have root URL that is either relative or absolute. Sites
    | are typically used for localization (eg. English/French) but may also
    | be used for related content (eg. different franchise locations).
    |
    */

    'sites' => [
        'xyz-com' => [
            'name' => 'xyz-com',
            'locale' => 'en_US',
            'url' => env('XYZSK_URL'),
        ],

        'xyz-pl' => [
            'name' => 'xyz-pl',
            'locale' => 'pl_PL',
            'url' => '/pl/',
        ],

        'xyz-support' => [
            'name' => 'xyz-support',
            'locale' => 'en_US',
            'url' => '/support/',
        ],

        'xyz-pomoc' => [
            'name' => 'xyz-pomoc',
            'locale' => 'pl_PL',
            'url' => '/pomoc/',
        ],

        'trainings' => [
            'name' => 'trainings',
            'locale' => 'en_US',
            'url' => '/',
        ]
    ],
];`
